src/aeb_fate_encoding.erl

```diff
@@ -270,8 +272,12 @@ deserialize_type(<<?TYPE_TUPLE, N, Rest/binary>>) ->
     {{tuple, Ts}, Rest2};
 deserialize_type(<<?TYPE_BYTES, Rest/binary>>) ->
     {N, Rest2} = deserialize_one(Rest),
-    true       = is_integer(N) andalso N >= 0,
-    {{bytes, N}, Rest2};
+    true       = is_integer(N),
+    if N == -1 ->
```

Comment:
Do any docs actually refer to this specifically chosen -1?

Reply:
Not yet - but it should go into `protocol` at some point.